content/boilerplates/notes/1.2.md

```diff
@@ -0,0 +1,64 @@
+## Installation
+
+- **Added** pod anti-affinity and toleration support to all of our control plane components.
+- **Added** `sidecarInjectorWebhook.neverInjectSelector` and `sidecarInjectorWebhook.alwaysInjectSelector` to allow users to further refine workloads that have sidecar automatically injected or not, based on label selectors.
+- **Added** `global.logging.level` and `global.proxy.logLevel` to allow users to easily configure logs for control plane and data plane components globally.
+- **Added** support to configure the Datadog location via `global.tracer.datadog.address`.
+
+## Upgrades
+
+- **Removed** deprecated Adapter and Template CRDs by default. Configure `mixer.templates.useTemplateCRDs=true` and `mixer.adapters.useAdapterCRDs=true` to re-enable them.
```

Comment:
@mandarjog 

This needs more exposition

Reply:
This is also described more in the upgrade notice:  https://github.com/istio/istio.io/pull/4471

Reply:
we could also link to the deprecation notice, if so desired: https://discuss.istio.io/t/deprecation-notice-custom-mixer-adapter-crds/2055

Reply:
added